dist/vue-carousel.js

```diff
@@ -717,6 +723,8 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	      if (this.modalEnabled) {
 	        width = this.browserWidth;
+	      } else if (this.showInline) {
+	        return page * this.nextOffset * -1;
```

Comment:
This seems a little hard to read and/or weird place for this (?) but I trust your judgement. Maybe we could come back and make it more readable later 😄